packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -771,7 +816,11 @@ export default decorate([
                         <Ul>
                           {map(state.remotes, (id, key) => (
                             <Li key={id}>
-                              <Remote id={id} />
+                              <Remote id={id} />{' '}
+                              <RemoteProxyWarning
```

Comment:
The warning should be displayed only if proxy is selected.
[If we don't select proxy, the warning will be displayed with `proxyId != null`](https://github.com/vatesfr/xen-orchestra/pull/4243/files#diff-9c044ab8a42ed6576ea927a64c1ec3ebR181)

Reply:
It's the case, the warning is not displayed when the proxy is not selected (proxy === null || proxy === undefined).